core/templates/components/state-editor/state-skill-editor/state-skill-editor.component.html

```diff
@@ -9,21 +9,21 @@
       <i class="fa fa-caret-up" *ngIf="skillEditorIsShown" aria-hidden="true"></i>
     </mat-card-actions>
   </mat-card-header>
-  <mat-card-actions class="oppia-skill-button-container" *ngIf="skillEditorIsShown && !stateLinkedSkillIdService.displayed">
+  <mat-card-actions class="oppia-skill-button-container" *ngIf="skillEditorIsShown && !isSkillSelected">
```

Comment:
If it's supposed to be a boolean, use skillIsSelected. Do not start variable names with verbs. This is a rule to follow in general -- see point 3 [here](https://github.com/oppia/oppia/wiki/Coding-style-guide#javascript).

Reply:
Done,